src/Neo/SmartContract/Native/HashIndexState.cs

```diff
@@ -9,20 +9,22 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
+#nullable enable
+
 using Neo.Extensions;
 using Neo.VM;
 using Neo.VM.Types;
 
 namespace Neo.SmartContract.Native
 {
-    class HashIndexState : IInteroperable
+    internal class HashIndexState : IInteroperable
```

Comment:
This file also does not have ANY comment. It is bassically just related to LedgerContract.cs only